oeps/oep-0002.rst

```diff
@@ -0,0 +1,101 @@
++---------------+-------------------------------------------+
+| Title         | Repository Metadata                       |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata for Open edX projects.
+
+Motivation
+==========
+
+One of the primary goals for Best Practice OEPs is to standardize techniques
+and tools across Open edX repositories. However, without
+tooling, it will be difficult to move all of the repositories in the same
+direction (and hard to validate which ones have been updated). This OEP exists
+to fill that gap.  By providing a standard format for metadata about accordance
+to Best Practice OEPs, tools can be developed in the future to report on the
+current status of all of the Open edX repositories, and how well they follow
+the Best Practices listed in the OEPs.
+
+Bear in mind that particular classes of Open edX repositories may have
+different Best Practices. The practices that apply to an XBlock may not be the
+same as those that apply to a standalone Django service or to a Django app.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``openedx.yml``, with the
+following fields:
+
+``owner``: string
+    The github user/team responsible for the repository.
+
+``nick``: string
+    A short-name for the repository used by reporting tools.
+
+``area``: string
+    Used to group repositories by in reporting tools. Sample areas: core,
+    mobile analytics.
+
+``oeps``: dictionary
+    A list of keys corresponding to Best Practice OEPs (in the format
+    ``oep-n``). Each value would be one of ``True``, ``False``, or a
+    reason dictionary. ``True`` and ``False`` would indicate accordance (or
+    lack thereof) with the specified OEP. A reason dictionary
+    must have ``state`` and ``reason`` keys. The ``state`` key is a boolean,
+    and ``reason`` is a string that should explain why it's ok that the
+    ``state`` is ``False`` (this will be displayed in reporting tools).
+
+    If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
+    ``False``, unless the reporting tool can auto-detect accordance.
+
```

Comment:
@cpennington it would be helpful to the reader if you included a chunk of sample YAML that shows all of these fields in action, with obviously fake values. Or you could just show snippets for each sub-section, whatever works for you, I just think a few examples would help illustrate the data types and enumerations.